### ISSUE TITLE
Typo: a11y:certifie_d_Credential should be a11y:certifie_r_Credential

### ISSUE THREAD

**Issue by danielweck** [daisy/kb#25]
http://kb.daisy.org/publishing/docs/metadata/evaluation.html#a11y:certifierCredential

**Comment by mattgarrish** [comment]
Thanks @danielweck. Fixed in https://github.com/daisy/kb/commit/108ce3f48a1ebf16231e01e80cbf7cba0fd27604
